syntax = "proto3";
option go_package = ".;animation";

enum AnimationID {
    SUCCESS = 0;
    ERROR = 1;
    LOADING = 2;
}

message AnimationArg {
    int32 duration = 1;
    AnimationID animation_id = 2;
}
